randovania/layout/description_migration.py

```diff
@@ -417,6 +417,31 @@ def _migrate_v22(data: dict) -> dict:
     return data
 
 
+def _migrate_v23(data: dict) -> dict:
+    game_modifications = data["game_modifications"]
+
+    for game in game_modifications:
+        game_name = game["game"]
+        if game_name != "am2r":
+            continue
+
+        migration = migration_data.get_raw_data(RandovaniaGame(game_name))["spazer_beam_typo"]
```

Comment:
Instead of using the migration data I'd just have this migration handle renaming that node in particular directly.

Migration data is mostly because certain migrations needed a lot of data for many games at once, but it's not reusable.

Reply:
I like migration data here, cause it means that if it's needed to rename nodes/areas in the future again, one can just copy paste it from here ~~like how I did from dread xP~~